### PR TITLE
Improve i18n

### PR DESCRIPTION
**Pull request by wernerkrauss**
Fix TextCheckboxGroupField translation
Translate settings tab
expose i18n Javascript

### CODE REVIEW

File: src/Models/BaseElement.php [hunk 1]
```diff
@@ -295,6 +295,10 @@ public function getCMSFields()
                     )
             );
 
+            // Rename the "Settings" tab
+            $fields->fieldByName('Root.Settings')
+                ->setTitle(_t(__CLASS__ . '.SettingsTabLabel', 'Content'));
```
Comment (robbieaverill) [inline src/Models/BaseElement.php]: You're renaming "Settings" to "Content"? Do you think that might be a little misleading?

Comment (wernerkrauss) [reply src/Models/BaseElement.php]: oops, copy/paste error, thanks for the heads up!

File: src/Models/BaseElement.php [hunk 2]
```diff
@@ -295,6 +295,10 @@ public function getCMSFields()
                     )
             );
 
+            // Rename the "Settings" tab
+            $fields->fieldByName('Root.Settings')
+                ->setTitle(_t(__CLASS__ . '.SettingsTabLabel', 'Settings'));
```
Comment (robbieaverill) [inline src/Models/BaseElement.php]: Ok, what was the label originally? Is this just so we can make the label translatable with an explicit translation key?

Comment (wernerkrauss) [reply src/Models/BaseElement.php]: The default "Settings"; I added the key to https://github.com/dnadesign/silverstripe-elemental/pull/578/files#diff-e0c0c7c5f72bb31c02647d33f6a0b563R17 according to the other label for "Content"

Is there a more elegant way for translating tab labels?

**Pull request merged**